CustomLogAppender/log4jconfig.yaml

```diff
@@ -0,0 +1,55 @@
+Configuration:
```

Comment:
the sample in the tutorial was yaml, but we need it in xml content:
fitting the log42 format -> the file should be called 'log4j2.xml'